### DISCUSSION TITLE
Is there is any way to limit text from pb

### DISCUSSION THREAD

**Discussion by MuhammadKhizar7** [pocketbase/pocketbase#2627] [Q&A] [answered]
Is there is any way to limit text from pockebase for example in blog card example we show just little amount of text on card not full blog content if not I would like to make a feature request as I think it's a common in most of  e-com, jobs, blogs, etc. listings,

**Comment by ganigeorgiev** [comment, marked as answer]
There is no builtin "excerpt"-like functionality. One alternative could be to create a View collection and make use of the SQLite [`substr` function](https://www.sqlite.org/lang_corefunc.html#substr), something like:


```sql
SELECT
    posts.id,
    substr(posts.content, 0, 100)
FROM posts
```

Another option if you have a dedicated excerpt/short description field is to use the `fields` query parameter (available with v0.16+) to explicitly set which fields to return:

```js
await pb.collection("example").getList(1, 30, {
    fields: "id,create,update,collectionId,shortDescription"
})
```

**Reply by MuhammadKhizar7** [reply]
Thanks
